ui/widgets/datepicker.js

```diff
@@ -172,6 +176,9 @@ var widget = $.widget( "ui.datepicker", {
 		},
 		blur: function() {
 			this.suppressExpandOnFocus = false;
+		},
+		change: function( event ) {
```

Comment:
Is the native change event reliable enough? Usually the answer to that is "no", may need to do a manual dirty-check in `blur` and trigger `change` from there.